ZipBuilder/Package.swift

```diff
@@ -21,12 +21,20 @@ import PackageDescription
 
 let package = Package(
   name: "ZipBuilder",
+  products: [
+      .library(name: "ManifestReader", targets: ["ManifestReader"]),
```

Comment:
I don't think we need to vend a library here - this should be an `executable` as well.

Reply:
Removed `products` section